content/en/docs/overview/singleton-sdk/Angular Client Introduction/i18n-context.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "I18n Context"
+date: 2022-06-16T12:43:05+03:00
+draft: false
+weight: 24
+---
+
+
+#### **Overview**
+
+I18n context is designed to manage the variables from different environments which carries pseudoEnabled, i18nEnabled, preferredLanguage, and other user-scoped values, provides a way to load and share these values between services without having to explicitly pass a parameter through every level of the service. In this way, users can use this interface to modify the default behavior regardless of the development or testing phase.
+The storage of these variables is configurable, and the default is saved in localstorage.
+
+```
+import { I18nContext } from "@singleton-i18n/angular-client";
+
+```
+
+#### **Context Attributes**
+
+|     Attribute     |  Type   | Writable | Default   | <div style="text-align:center">Description</div>                                                                                                                                                                    |
+|:-----------------:|:-------:|----------|-----------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|   pseudoEnabled   | boolean | readonly | undefined | The pseudo setting from localStorage, determine whether to show pseudo translation in debugging or developing stage. The value can be changed by localStorage, it will override the pseudo setting in i18n configs. |
+|    i18nEnabled    | boolean | writable | undefined | The i18n enable setting from localStorage, determine whether to use vip features in testing or product stage.                                                                                                       |
```

Comment:
i18nEnabled    | boolean | writable | undefined | The i18n enable setting from localStorage, determine whether to use Singleton features in testing or product stage.

Reply:
Done!